frameworks/file-analysis.rst

```diff
@@ -90,34 +90,69 @@ Adding Analysis
 ===============
 
 Zeek supports customized file analysis via `file analyzers` that users can
-attach to observed files.  Once attached, file analyzers start receiving the
-contents of files as Zeek extracts them from ongoing network connections.
+attach to observed files. You can attach analyzers selectively to individual
+files, or register them for auto-attachment under certain conditions. Once
+attached, file analyzers start receiving the contents of files as Zeek extracts
```

Comment:
Could we avoid "extract" here steer clear of the confusion with the file-extraction part implying persistent storage? Maybe "observe" as you used in the file extraction example further down?

```suggestion
attached, file analyzers start receiving the contents of files as Zeek observes them on the network.
```

Reply:
I wondered about that too. :-) I initially used "parsed" and then reverted it, trying to minimize changes. I'll tweak it.